### ISSUE TITLE
delins cause Exception in thread "main" java.lang.StringIndexOutOfBoundsException: String index out of range: 0

### ISSUE THREAD

**Issue by leipzig** [charite/jannovar#87]
Not sure if you really want to accept delins (what caller would produce them?) but the error should at least tell you what line to remove from your vcf
```
##fileformat=VCFv4.1
##source=HGMD_PRO_2013.4
##version=0.1_beta
##reference=GRCh37
##fileDate=20131215
##comment="REF and ALT sequences are both on forward strand of reference assembly"
##INFO=<ID=CLASS,Number=1,Type=String,Description="Mutation Category, https://portal.biobase-international.com/hgmd/pro/global.php#cats ">
##INFO=<ID=MUT,Number=1,Type=String,Description="HGMD mutant allele">
##INFO=<ID=GENE,Number=1,Type=String,Description="Gene symbol">
##INFO=<ID=STRAND,Number=1,Type=String,Description="Gene strand">
##INFO=<ID=DNA,Number=1,Type=String,Description="DNA annotation">
##INFO=<ID=PROT,Number=1,Type=String,Description="Protein annotation">
##INFO=<ID=DB,Number=1,Type=String,Description="dbSNP identifier, build 137">
##INFO=<ID=PHEN,Number=1,Type=String,Description="HGMD primary phenotype">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
X	103041654	CX136110	AGGTGATC	AA	.	.	CLASS=DM;MUT=ALT;GENE=PLP1;STRAND=+;DNA=NM_000533.3:c.453_453+6delGGTGATCinsA;
```

**Comment by holtgrewe** [comment]
OK, I could reproduce the problem with `annotate-pos data/hg19_ucsc.ser 'chrX:103041654AGGTGATC>AA'`.

These variants should work and also the program should not crash in these cases but just print a warning. I will fix this.

**Comment by holtgrewe** [comment]
OK, this should be fixed in develop after merging #90 and will be part in the `v0.12` release.
